recipes/intake-dremio/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "intake-dremio" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/intake-dremio-{{ version }}.tar.gz
+  sha256: 2cfaebe5671592c40c19c6a77e0c63eff68b23ebdfbd767f6c96967c359bca6d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - intake
+    - pandas
+    - python >=3.7
+    - pyarrow >=4.0
+
+test:
+  imports:
+    - intake_dremio
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/intake/intake-dremio
```

Comment:
A minor remark: the `setup.py` still refers to <https://github.com/philippjfr/intake-dremio>: <https://github.com/intake/intake-dremio/blob/41832f6e61de72dc9db3a2b0a00ca97153f97ebe/setup.py#L33>

There is a redirect, but it might be good to update it. No impact on the conda recipe.